simexpal/launch/common.py

```diff
@@ -453,9 +468,16 @@ def close(self):
 		# Consume any output that might be ready.
 		events = sel.select(timeout=1)
 		for (sk, mask) in events:
-			if not sk.data.progress():
+			if sk.fileobj == signal_reader:
+				sel.unregister(signal_reader)
```

Comment:
Why do we need to unregister `signal_reader` here? It probably makes more sense to just clear the status by reading from the pipe.

Reply:
We have to unregister the ``signal_reader`` at some point, so I think after receiving the first signal, we no longer need to listen to it